pages/common/nmap.md

```diff
@@ -1,37 +1,37 @@
 # nmap
 
 > Network exploration tool and security / port scanner.
-> Some features only activate when Nmap is run with privileges.
+> Some features only activate when Nmap is run with root privileges.
 > More information: <https://nmap.org>.
 
 - Check if an IP address is up, and guess the remote host's operating system:
 
 `nmap -O {{ip_or_hostname}}`
 
-- Try to determine whether the specified hosts are up and what their names are:
+- Try to determine whether the specified hosts are up (ping scan) and what their names are:
 
 `nmap -sn {{ip_or_hostname}} {{optional_another_address}}`
 
-- Like above, but also run a default 1000-port TCP scan if host seems up:
-
-`nmap {{ip_or_hostname}} {{optional_another_address}}`
-
 - Also enable scripts, service detection, OS fingerprinting and traceroute:
 
 `nmap -A {{address_or_addresses}}`
 
-- Assume good network connection and speed up execution:
+- Scan a specific list of ports (use '-p-' for all ports from 1 to 65535):
+
+`nmap -p {{port1,port2,…,portN}} {{address_or_addresses}}`
 
-`nmap -T4 {{address_or_addresses}}`
+- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results ('-oN') to output file:
 
-- Scan a specific list of ports (use `-p-` for all ports `1-65535`):
+`nmap -sC -sV -oN {{top-1000-ports.txt}} {{address_or_addresses}}`
 
-`nmap -p {{port1,port2,…,portN}} {{address_or_addresses}}`
```

Comment:
I'd argue that this is a useful command to have

Reply:
^ the previous one

Reply:
@CleanMachine1 I am bit confused, you mean this one is useful:

```
nmap -T4 {{address_or_addresses}}
```

Reply:
I believe @CleanMachine1 means this example:
```markdown
- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results ('-oN') to output file:

`nmap -sC -sV -oN {{top-1000-ports.txt}} {{address_or_addresses}}`
```

Reply:
`nmap -sC -sV -oN {{top-1000-ports.txt}} {{address_or_addresses}}` is a very useful command used by most professional pentesters and CTF players. Please watch one of ippsec's hacking a box demo here https://www.youtube.com/watch?v=4e4wKDrANog 

(IppSec is a well respected individual in the inforsec space)